content/package.json

```
@@ -0,0 +1,5 @@
+{
+    "dependencies": {
+        "require-glob": "^4.1.0"
+    }
+}
```

```suggestion
{
    "dependencies": {
        "require-glob": "^4.1.0"
    },
    "workspaces": {
        "build",
        "tutorials"
    }
}
```

I'd like to suggest to define the workspaces here to keep the "tree structure" consistent. See corresponding update to root level `package.json`.

---

Actually now I think of it we can remove the `package.json` files in build and tutorials folders and add `*/*/*/documentation` and `*/*/documentation` here.

---

Actually we can't. I thought about it. But maybe I'm overseeing something. But we also have none `documentation` workspaces and a different level of depth in `tutorials`. Am I missing something?

---

In case of non-`documentation` workspaces we need to add the workspace explicitly anyways, because we can't do `*/*` or similar if some of the projects are nodejs projects themselves.

---

Which is another reason why it might be worth another look at how we utilize git to checkout projects, because if we can sparse checkout only documentation directories, that problem would go away and our logic would reduce to a single recursive glob. I think latest changes of git actually allow this, but I have to check if GitHub implemented that yet.